contracts/facets/TermFacet.sol

```diff
@@ -159,6 +173,62 @@ contract TermFacet is ITerm {
         }
     }
 
+    function _joinTerm(uint _termId, bool _optYield, uint _position) internal {
```

Comment:
I believe you also renamed the internal function, you can keep this internal function name to be _joinTermByPosition. The previous comment about keeping the same function name was only for the external functions for a cleaner ABI

Reply:
Commit 7dffd34 and tests on commit 810b5a8

Reply:
I mistake comments, actually I did not read this one. This commits are for the previous comment

Reply:
Commit 5fbf37b renamed again the internal one for better readability.